inmem.go

```diff
@@ -120,6 +121,27 @@ func (a *AggregateSample) String() string {
 	}
 }
 
+// NewInmemSinkFromURL creates an InmemSink from a URL. It is used
+// (and tested) from NewMetricSinkFromURL.
+func NewInmemSinkFromURL(u *url.URL) (MetricSink, error) {
+	params := u.Query()
+	if len(params["interval"]) < 1 {
```

Comment:
You can use `.Get(key)` on a `url.Values` instead of checking length + indexing to the 0th thing. I would just do that and check against `""` to make sure it was set.

https://golang.org/pkg/net/url/#Values.Get